src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Arm64/Aes.PlatformNotSupported.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
```

Comment:
We try and not add an extra line after this to whenever we want to flip the switch to the whole assembly make the grep/replace easier, but I guess it is doable this way as well.

Reply:
Fixed. I had just gone with what felt natural/readable 😄